po/en.po

```diff
@@ -1778,8 +1778,10 @@ msgstr "Illegal file descriptor in redirection “%ls”"
 
 #: parse_constants.h:213
 #, c-format
-msgid "No matches for wildcard '%ls'."
-msgstr "No matches for wildcard “%ls”."
+msgid "No matches for wildcard pattern '%ls'.\n"
+"(Tip: To avoid wildcarding, quote it. To allow no matches, expand it to a variable.)"
```

Comment:
It's because "expand it to a variable" is grammatically awkward and therefore likely to confuse some users.  I can't find a single instance of "wildcard pattern" in the docs or source. There are 36 lines in *doc_src* containing "wildcard" and 40 containing "glob". So if you want to say "No matches for wildcard '%ls'.\n" that's fine by me. I'd also replace "wildcarding" with "wildcard expansion".